pkg/vm/engine/disttae/database.go

```diff
@@ -0,0 +1,66 @@
+// Copyright 2022 Matrix Origin
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package disttae
+
+import (
+	"context"
+
+	"github.com/matrixorigin/matrixone/pkg/catalog"
+	"github.com/matrixorigin/matrixone/pkg/vm/engine"
+)
+
+var _ engine.Database = new(database)
+
+func (db *database) Relations(ctx context.Context) ([]string, error) {
+	return db.txn.getTableList(ctx, db.databaseId)
+}
+
+func (db *database) Relation(ctx context.Context, name string) (engine.Relation, error) {
+	id, err := db.txn.getTableId(ctx, db.databaseId, name)
+	if err != nil {
+		return nil, nil
+	}
+	return &table{
+		tableId:   id,
+		tableName: name,
+		db:        db,
+	}, nil
+}
+
+func (db *database) Delete(ctx context.Context, name string) error {
+	if err := db.txn.WriteBatch(DELETE, catalog.MO_CATALOG_ID, catalog.MO_TABLES_ID,
```

Comment:
Why is this WriteBatch?   Seems we are writing one by one?

Reply:
Yes, but the mergeentry function will be added later.